Documentation/ceph-cluster-crd.md

```diff
@@ -930,3 +908,43 @@ spec:
   cephVersion:
     image: ceph/ceph:v14.2.9 # Should match external cluster version
 ```
+
+### Cleanup policy
+
+Rook has the ability to cleanup resources and data that were deployed.
+The policy represents the confirmation that cluster data should be forcibly deleted.
+The cleanupPolicy should only be added to the cluster when the cluster is about to be deleted.
+After any field of the cleanup policy is set, Rook will stop configuring the cluster as if the cluster is about to be destroyed in order to prevent these settings from being deployed unintentionally.
+The `cleanupPolicy` CR settings has different fields:
+
+* `confirmation`: If `yes-really-destroy-data` the operator will automatically delete data on the hostpath of cluster nodes and clean devices with OSDs when a `delete cephcluster` command is issued. Only `yes-really-destroy-data` and an empty string are valid values for this field.
+* sanitizeDisks: sanitizeDisks represents advanced settings that can be used to sanitize drives.
+By default Rook, will only remove Ceph's metadata, which allows a cluster re-installation if desired.
+However, the administrator might want to sanitize the drives in more depth, that's what the following flags do:
+  * method: indicates if the entire disk should be sanitized or simply ceph's metadata. Possible choices are 'data' or 'metadata' (default)
+  * utility: indicates the tool to use to sanitize the disk. Possible choices are 'dd' (default) or 'shred'.
```

Comment:
Why do we need to expose the command line tool in the CR? This is an implementation detail of the policy. Can't we figure out internally which tool to use based on the settings?

Reply:
it's probably doable, but exposing the `utility` leaves the door open to more utilities. That's the intent.

Reply:
I still don't see the need to expose the command line tool. If another utility is needed, it would be driven by some other settings in the CR.

Reply:
Fixed!